packages/website/ts/pages/explore/explore_content.tsx

```diff
@@ -0,0 +1,138 @@
+        url: 'https://radarrelay.com/',
+        keywords: ['relayer'],
+        instant: {
+            orderSource: 'https://api.radarrelay.com/0x/v2/',
+        },
+    },
+    emoon: {
+        name: 'emoon',
+        label: 'Emoon',
+        description: 'Emoon is a peer-to-peer marketplace for the exchange of ERC-20 and ERC-721 crypto assets.',
+        logo_url: '/images/explore/emoon.png',
+        theme_color: '#3F89E7',
+        url: 'https://www.emoon.io/',
+        keywords: ['relayer', 'collectibles'],
+    },
+    openrelay: {
+        name: 'openrelay',
+        label: 'OpenRelay',
+        description:
+            'Open Relay is an open order book relayer with a focus on scalable and open source backend infrastructure.',
+        logo_url: '/images/explore/open_relay.png',
+        theme_color: '#163AAB',
+        url: 'https://openrelay.xyz/',
+        keywords: ['relayer'],
+    },
+    boxswap: {
+        name: 'boxswap',
+        label: 'BoxSwap',
+        description: 'OTC relayer made for swapping ERC-20 and ERC-721 assets in a marketplace communities.',
+        logo_url: '/images/explore/box_swap.png',
+        theme_color: '#FF99DF',
+        url: 'https://boxswap.io/',
+        keywords: ['collectibles'],
+    },
+};
+
+export const EDITORIAL: ExploreTile[] = [
+    {
+        name: 'editorial_featured',
+        component: (
+            <ExploreGridFeaturedTile
+                {...PROJECTS.paradex}
+                headerImageUrl={'/images/explore/editorial/featured/paradex.png'}
+            />
+        ),
+        visibility: ExploreTileVisibility.Visible,
+        width: ExploreTileWidth.FullWidth,
+    },
+];
+
+export const FILTERS: ExploreFilterMetadata[] = [
+    {
+        label: 'All',
+        name: 'all',
+        filterType: ExploreFilterType.All,
+    },
+    {
+        label: 'Relayer',
+        name: 'relayer',
+        filterType: ExploreFilterType.Keyword,
+    },
+    {
+        label: 'Collectibles',
+        name: 'collectibles',
+        filterType: ExploreFilterType.Keyword,
+    },
+];
+
+export const ORDERINGS: ExploreTilesOrderingMetadata[] = [
+    { label: 'Popular', ordering: ExploreTilesOrdering.Popular, type: ExploreTilesOrderingType.HardCodedByName },
+    {
+        label: 'Recently Added',
+        ordering: ExploreTilesOrdering.RecentlyAdded,
+        type: ExploreTilesOrderingType.HardCodedByName,
+    },
+    {
+        label: 'Alphabetical',
+        ordering: ExploreTilesOrdering.Alphabetical,
+        type: ExploreTilesOrderingType.HardCodedByName,
+    },
+];
+
+export const BY_NAME_ORDERINGS: { [s: string]: string[] } = {
+    [ExploreTilesOrdering.Popular]: ['boxswap', 'veil', 'paradex', 'emoon', 'radar_relay', 'openrelay'],
+    [ExploreTilesOrdering.RecentlyAdded]: ['veil', 'boxswap', 'emoon', 'paradex', 'radar_relay', 'openrelay'],
+    [ExploreTilesOrdering.Alphabetical]: ['veil', 'boxswap', 'emoon', 'paradex', 'radar_relay', 'openrelay'],
```

Comment:
ah. Can't we just do this programmatically. Maybe for the Recently added as well? (Assume natural order is added order).